mantidproject/mantid
# Update GUI to use user inputted values if applicable.

**Description of work.**

Currently the new Muon GUI uses the values for timezero, first good time and deadtimetable stored in the file and ignores and selection made by the user in the GUI. This PR updates the GUI to use these user inputted values if applicable.

**Report to:** Anthony

**To test:**

Review the code and try out the options on the home tab for updating the values. Check that values of the outputed groups change. Note that currently you need to manually click update all to propogate these changes.



Fixes #24647 



*This does not require release notes* because the new GUI is not yet user facing


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
